recipes/libelf/all/conanfile.py

```diff
@@ -83,6 +83,7 @@ def generate(self):
 
     def build(self):
         if self.settings.os == "Windows":
+            replace_in_file(self, os.path.join(self.source_folder, "lib", "version.c"), "#include <private.h>", "#include <private.h>\n#include <stdlib.h>")
```

Comment:
Could you fix this through a patch file instead please?

Reply:
@SpaceIm Thank's for your response. It's my first time contributing to this repo. I'll try to provide a patch file.

Reply:
Replaced by patch file!